phpBB/assets/javascript/core.js

```diff
@@ -263,7 +263,11 @@ phpbb.ajaxify = function(options) {
 			}
 			phpbb.clearLoadingTimeout();
 			var errorText = false;
-			if (typeof errorThrown === 'string' && errorThrown.length > 0) {
+			var responseText = $.parseJSON(jqXHR.responseText);
```

Comment:
[From the doc](http://api.jquery.com/jQuery.parseJSON/):

> Passing in a malformed JSON string results in a JavaScript exception being thrown.

So I think the exception should be handled here somehow.